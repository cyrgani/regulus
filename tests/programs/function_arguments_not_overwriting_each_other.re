# old bug: used to output 2 2, correct is 2 1
def(foo, a, b, print(a, b)),

def(bar, a, b, _(
    print("a is", a, "b is", b),
    foo(b, a),
)),

bar(1, 2),
