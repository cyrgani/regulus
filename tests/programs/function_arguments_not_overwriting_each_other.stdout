a is 1 b is 2 
2 1 
